curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60fabf0dd4959805dbae09e6.md

```diff
@@ -9,7 +9,7 @@ dashedName: step-31
 
 Add another `label` after the first, with the text `Input your age (years): `. Then, nest an `input` with the `type` of `number`.
 
-As we do not want users under the age of 13 to register, add a `min` attribute to the `input` with a value of `13`. Also, we can probably assume users over the age of 120 will not register; add a `max` attribute with a value of `120`.
+Next, add a `min` attribute to the `input` with a value of `13` because users under the age of 13 should not register. Also, it can be probably assumed that users over the age of 120 will not register; add a `max` attribute with a value of `120`.
```

Comment:
mild edit for simplicity?
```suggestion
Next, add a `min` attribute to the `input` with a value of `13` because users under the age of 13 should not register. Also, users probably will not be over the age of 120; add a `max` attribute with a value of `120`.
```

Reply:
Thanks for your suggestion of simplified instruction text.